test/wasm2asm/i64-shifts.wast

```diff
@@ -0,0 +1,81 @@
+;; Testing i64 shifts
```

Comment:
Please run `./auto_update_tests.py` and add the output file it creates. That will be used by the test suite going forward.